build/cli.js

```diff
@@ -21,7 +22,87 @@ const {
 const { uniqifyTranslationsOf } = require("./translationsof");
 const { humanFileSize } = require("./utils");
 
-async function buildDocuments(files = null, quiet = false) {
+async function buildDocumentInteractive(
```

Comment:
It's a bit weird that it's called `buildDocumentInteractive` but it's only *maybe* interactive. :)

Reply:
Ignore this. It's just a punny notice.